src/liquid/components/ld-badge/ld-badge.tsx

```diff
@@ -0,0 +1,77 @@
+import { Component, Element, h, Host, Prop, State, Watch } from '@stencil/core'
+import { HTMLStencilElement } from '@stencil/core/internal'
+import { getClassNames } from 'src/liquid/utils/getClassNames'
+
+/**
+ * @virtualProp ref - reference to component
+ * @virtualProp {string | number} key - for tracking the node's identity when working with lists
+ */
+@Component({
+  assetsDirs: ['assets'],
+  tag: 'ld-badge',
+  styleUrl: 'ld-badge.css',
+  shadow: true,
+})
+export class LdBadge {
+  @Element() el: HTMLStencilElement
+
+  /** Defines badge custom color */
+  @Prop() brandColor?: boolean
+
+  /** Icon name. */
+  @Prop() icon?: string
+
+  /** The size of the badge, translated in rem */
+  @Prop() size?: 'sm' | 'lg'
```

Comment:
```suggestion
  @Prop() size?: 'lg'
```

At least, there seems to be no "--size-sm" in the CSS.